quiz/docker-compose.yaml

```diff
@@ -0,0 +1,38 @@
+services:
+  backend_quiz:
+    build:
+        context: ./backend
+    command: python manage.py runserver 0.0.0.0:8000
+    env_file:
+      - ./backend/.envs/.env
+      - ./.envs/.env_db
+    ports:
+      - 8000:8000
```

Comment:
Na masterze jest już proxy dla dev & prod, przydałoby się wpiąć apke w to.

Reply:
dograłam mastera i właśnie staram się to ogarnąć

Reply:
ogólnie wszystko co było u mnie zamieniam na wersję z mastera